homeassistant/components/goalzero/sensor.py

```diff
@@ -75,7 +75,6 @@
         name="WH Stored",
         device_class=SensorDeviceClass.ENERGY,
         native_unit_of_measurement=ENERGY_WATT_HOUR,
-        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
This is not necessarily wrong.
If the sensor represents the charge in Wh held by a battery or something like that, we should not change this.

Reply:
Thanks for confirmation - and sorry for the misunderstanding.